core/src/main/java/com/homeaway/streamplatform/streamregistry/exceptions/InvalidClusterException.java

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2018 Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamplatform.streamregistry.exceptions;
+
+import com.homeaway.streamplatform.streamregistry.model.JsonCluster;
+
+public class InvalidClusterException extends RuntimeException{
+
+    protected final JsonCluster cluster;
+
```

Comment:
just a opinion. we can extend Exception, so that it has to be thrown and mentioned in the Interface as well.
in that way, client would know that they have to take action on this exception.